fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/patient/ChunkReader.java

```diff
@@ -154,12 +156,20 @@ protected void fillChunkDataBuffer(List<String> patientIds) throws Exception {
                 if (!searchCriteria.isEmpty()) {
                     queryParameters.put(Constants.FHIR_SEARCH_LASTUPDATED, searchCriteria);
                 }
-
                 queryParameters.put("_sort", Arrays.asList(new String[] { Constants.FHIR_SEARCH_LASTUPDATED }));
 
-                for (String patientId : patientIds) {
+                List<String> compartmentSearchCriterias = CompartmentUtil.getCompartmentResourceTypeInclusionCriteria("Patient", resourceType.getSimpleName());
+                if (compartmentSearchCriterias.size() > 1) {
+                    isDoDuplicationCheck = true;
```

Comment:
where is this used?  does search return the same resource multiple times (one for each include) and is that a bug?

Reply:
used in following resources loop, because _typeFilters, and compartmentsearchCriterias search results can have duplication, this was why the duplication removing codes was added..